### ISSUE TITLE
Getting "Failed to determine git tag" when pushing

### ISSUE THREAD

**Issue by no2chem** [replicate/cog#1093]
I'm running on macos. I installed cog by getting the prebuilt binary.

I wanted to try pushing a clone of the bark repository. So I ran:

```sh
cog login
git clone https://github.com/chenxwh/bark
cd bark
cog push r8.im/no2chem/<repo-name>
```

The container builds, but the push fails after the image is built:
```
Adding labels to image...
⚠ Failed to determine Git tag: exit status 128
```

The label is never added, causing all subsequent steps to fail.,

**Comment by anotherjesse** [comment]
@no2chem interesting - I have a similar issue with "failed to determine git tag" but it succeeds in pushing anyway.

(I'm assuming the duplicate cog in `cog cog push r8.im/no2chem/<repo-name>` is a typo from the bug report?)

**Comment by no2chem** [comment]
yes the duplicate `cog` is a typo. 

It seems like it isn't the root cause though, I see that error on my x86 linux machine.

```
#1 [internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.0s

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 107B done
#2 DONE 0.0s

#3 [internal] load metadata for r8.im/no2chem/<repo-name>:latest
#3 ...

#4 [auth] no2chem/<repo-name>:pull replicate-production/replicate-us/no2chem/<repo-name>:pull token for r8.im
#4 DONE 0.0s

#3 [internal] load metadata for r8.im/no2chem/<repo-name>:latest
#3 ERROR: r8.im/no2chem/<repo-name>:latest: not found
------
 > [internal] load metadata for r8.im/no2chem/<repo-name>:latest:
------
Dockerfile:1
--------------------
   1 | >>> FROM r8.im/no2chem/<repo-name>
--------------------
error: failed to solve: r8.im/no2chem/<repo-name>: r8.im/no2chem/<repo-name>: not found

ⅹ Failed to add labels to image: exit status 1
``

**Comment by scotm** [comment]
Hi,

I resolved this error on my own branch/model - by adding a tag to the current commit to the repository.

`git tag -a 0.0.1`

**Comment by mattt** [comment]
Hi @no2chem. As @anotherjesse suggested, the warning you're seeing about not being able to determine the Git tag is a red herring. Contrary to its warning severity, that error is non-fatal, and the failure has to do with `ERROR: r8.im/no2chem/<repo-name>:latest: not found`.

I just opened #1111 to improve how all of this is communicated.

The root cause of your build failure may have to do with a regression that was fixed by #1097. When you have a chance, please download the latest version (v0.8.0-beta5) and let me know if that resolves your issue.
